durabletask/aio/internal/shared.py

```diff
@@ -40,11 +51,19 @@ def get_grpc_aio_channel(
             host_address = host_address[len(protocol) :]
             break
 
+    # channel interceptors/options
+    channel_kwargs: Dict[str, ChannelArgumentType | Sequence[ClientInterceptor]] = dict(
+        interceptors=interceptors
+    )
+    if options is not None:
+        validate_grpc_options(options)
+        channel_kwargs["options"] = options
+
     if secure_channel:
         channel = grpc_aio.secure_channel(
-            host_address, grpc.ssl_channel_credentials(), interceptors=interceptors
+            host_address, grpc.ssl_channel_credentials(), **channel_kwargs
         )
```

Comment:
Can't we just do this? I don't see why we need to build `kwargs` with interceptors and options, we can just send `None` if options is set to `None`, no?

```suggestion
    if options is not None:
        validate_grpc_options(options)
    
    if secure_channel:
        channel = grpc_aio.secure_channel(
            host_address,
            grpc.ssl_channel_credentials(),
            interceptors=interceptors,
            options=options,
        )
```

Reply:
sure. BTW,  I could not commit your suggestion as github complained about "cannot commit on deleted lines) or something like it. I will put in another PR with all the other feedback tackled